clocks/build.gradle

```diff
@@ -0,0 +1,51 @@
+apply plugin: 'java-library'
+apply plugin: 'kotlin'
+apply plugin: 'jacoco'
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    api "io.reactivex.rxjava3:rxjava:3.0.0-RC3"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+
+    testImplementation 'junit:junit:4.12'
+    testImplementation 'org.hamcrest:hamcrest-library:1.3'
+}
+buildscript {
+    ext.kotlin_version = '1.3.50'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+repositories {
+    mavenCentral()
+}
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
+/*
+task jacocoTestReport(type: JacocoReport) {
+    group "Reporting"
+    description "Generate Jacoco coverage reports."
+
+    reports {
+        xml.enabled = false
+        html.enabled = true
+    }
+    getSourceDirectories().setFrom(files(["src/main/java"]))
+    getClassDirectories().setFrom((file("$project.buildDir/intermediates/classes/debug").exists()
+        ? fileTree(dir: "$project.buildDir/intermediates/classes/debug")
+        : fileTree(dir: "$project.buildDir/classes")))
+    getExecutionData().setFrom(fileTree(dir: "$project.buildDir", includes: ["jacoco/*.exec"]))
+}
+*/
```

Comment:
Is this task necessary or could it be useful in the future? If not, the comment could be removed. What do you think? ✂️

Reply:
This pull request is a draft 😅 I'm struggling with jacoco tasks to make it multimodule.
I made the draft to be able to launch travis and test the changes i'm introducing :)

Reply:
oops... I didn't pay attention to the draft state u.u'

Reply:
@all-contributors please add @tbenning for review

Reply:
@all-contributors please add @SergiLlamasGuerrero  for review

Reply:
Deleted!